CHANGE.txt

```diff
@@ -2,6 +2,14 @@
 LabKey Python Client API News
 +++++++++++
 
+What's New in the LabKey 2.6.2 package
```

Comment:
In other client API languages, we considered this a major version upgrade since it's not backwards compatible with old servers.

Reply:
Yea, I debated that but wanted to see what others thought. Fine pushing to 3.0.0.

Reply:
Bumped.